src/library_wasmfs.js

```diff
@@ -25,18 +27,13 @@ var WasmfsLibrary = {
           if (onload) onload();
           removeRunDependency(dep);
         }
-        var handled = false;
-        Module['preloadPlugins'].forEach(function(plugin) {
-          if (handled) return;
-          if (plugin['canHandle'](fullname)) {
-            plugin['handle'](byteArray, fullname, finish, function() {
-              if (onerror) onerror();
-              removeRunDependency(dep);
-            });
-            handled = true;
-          }
-        });
-        if (!handled) finish(byteArray);
+        if (Browser.handledByPreloadPlugin(byteArray, fullname, finish, function() {
```

Comment:
You can use `() =>` now rather than `function()`

Reply:
Ok, thanks. I'll leave that for later as just doing it here would cause an asymmetry in the similar code in FS, I'll do both as a followup.

Reply:
sgtm